<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfoDialog</class>
 <widget class="QDialog" name="FileInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>259</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelArtist">
       <property name="styleSheet">
        <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Исполнитель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditArtist">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTrackName">
       <property name="styleSheet">
        <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTrackName"/>
     </item>
     <item>
      <widget class="QLabel" name="labelAlbum">
       <property name="styleSheet">
        <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Альбом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAlbum"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPublisher">
       <property name="styleSheet">
        <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Издатель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPublisher"/>
     </item>
     <item>
      <widget class="QLabel" name="labelCopyright">
       <property name="styleSheet">
        <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Авторские права</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCopyright"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelGenre">
         <property name="styleSheet">
          <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Жанр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGenre"/>
       </item>
       <item>
        <widget class="QLabel" name="labelYear">
         <property name="styleSheet">
          <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Год</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditYear"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelDir">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDirTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Расположение</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFileNameTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Имя файла</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameFile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeFileTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Размер</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeFile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateSetTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Дата создания</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateSet">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodecTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Кодек</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodec">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBitrateTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Битрейт</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBirate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChannelCountTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Количество каналов</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChannelCount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSampleRateTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Частота дискретизации</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSampleRate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
